doc/swagger-api-description/api.yaml

```diff
@@ -245,6 +245,195 @@ paths:
         '404':
           description: The specified cloud account was not found.
 
+  /vufs/snap/{userUuid}:
+    get:
+      description: Allows to get snapshot vufs.
+      operationId: getVUFS
+      parameters:
+        - in: path
+          name: userUuid
+          description: Current user UUID.
+          required: true
+          type: string
+      responses:
+        '200':
+          description: VUFS snapshot of current user.
+          schema:
+            $ref: "#/definitions/Vufs"
+        '404':
+          description: Bad user UUID.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+        default:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+
+  /vufs/snap/diff/{userUuid}:
+    get:
+      description: Allows to get difference of previous snapshot and actual.
+      operationId: getDiffVUFS
+      parameters:
+        - in: path
+          name: userUuid
+          description: Current user UUID.
+          required: true
+          type: string
+      responses:
+        '200':
+          description: Difference VUFS snapshot of current user.
+          schema:
+            $ref: "#/definitions/Vufs"
+        '404':
+          description: Bad user UUID.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+        default:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+    post:
+    #QUESTION: Can we create VUFS snapshot with changes in local file storage on client side? Or we send a 'signal' about changes and server check what exactly has changed. And as response send VUFS diff object or full VUFS object?
```

Comment:
Diff can make client and server.
Case 1 - If when client receive event from local file storage monitoring services, VUFS generate diff. But this diff contain files not upload on clouds. Client send request with the diff to server and get response from server with the diff contains advanced allocate information  (where server write allocate space for each file part).
Case 2 - If server receive event from cloud monitoring about change files, VUFS on server make diff contains changes and send him on clients as push notification.